content/en/blog/2023-11-modelix-platform-release-23.2/index.md

```diff
@@ -0,0 +1,89 @@
+
+## Model client as NPM package (alpha)
+
+The previously existing [model client](https://github.com/modelix/modelix.core/tree/modelix-23.2/model-client) is now also published as [`@modelix/model-client` NPM package](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/).
+
+The exposed API is minimal and experimental. In the future, we want to make more features from the Kotlin model client library directly accessible. 
+
+## Vue.js Bindings (alpha)
+
+Built upon [`@modelix/model-client`](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/) we created a library to interact with models through Vue.js.
+
+With [`@modelix/vue-model-api`](https://docs.modelix.org/modelix/modelix-23.2/core/reference/component-vue-model-api.html) we are currently building sample projects with Vue.js. After that, we want to stabilize the API and show you how you can use it for your projects.
+
+## MPS Model Adapters
+We created [adapters](https://github.com/modelix/modelix.core/tree/modelix-23.2/mps-model-adapters) that serve as an interface between MPS and the modelix model-api.
+They are compatible with all MPS versions that modelix currently supports.
+
+
+## Bulk Model Synchronization Plugin (alpha)
+The [bulk-model-sync Gradle Plugin](https://docs.modelix.org/modelix/modelix-23.2/core/reference/component-bulk-model-sync-gradle.html) allows bulk synchronization of models between MPS and a model-server.
+Changes are applied incrementally and it can be run in existing CI systems to schedule and run the synchronization in both directions.
+This way you can implement a custom Git Integration tailored to your needs.
+
+The plugin is currently considered unstable, but will be stabilized in the coming weeks.
+
+# Development Insights
+
+## New approach to MPS related components
+
+Up until now, we built and released MPS specific components for each MPS version individually.
+Going forward, we want to release single components, which are compatible with all MPS versions, that are currently supported by modelix.
+The first component to follow this new approach is [mps-model-adapters](#mps-model-adapters)
+
+To achieve that, we separate code in MPS solutions from support library code wherever possible.
+Also, we are developing upcoming MPS plugins as IntelliJ plugins.
+
+## Contribution guide
+
+We created a [contribution guide](https://docs.modelix.org/modelix/main/main/contribute.html) to guide onboarding and external contributors.
+
+## Marking experimental features
+
+We decided to mark features sometimes as experimental so that we could publish them but still iterate quickly with ideas and APIs.
+
+A separate blog post will be published detailing the annotations used in Kotlin and TypeScript that mark features as experimental.
```

Comment:
Why do we actually need this blog post? As a reader I'd directly want to read this now. We can easily link this section to anyone interested. Or both this entry and the other one are published in parallel to make this section linkable.

Reply:
Writing a separate blog post is overkill.
I ditched the separate post https://github.com/modelix/website/pull/9 and linked the documentation here.